{
    "id": "f435f5756eed9e6e6b4a5d269a441e14274262b8726205ede209d501a1549c84",
    "title": "Sycophancy in GPT-4o: What happened and what we're doing about it",
    "link": "https://openai.com/index/sycophancy-in-gpt-4o/",
    "published_iso": "2025-04-30T03:06:26",
    "summary": "<a href=\"https://news.ycombinator.com/item?id=43840842\">Comments</a>",
    "source_feed": "https://news.ycombinator.com/rss",
    "scraped_at_iso": "2025-04-30T03:48:03.325740"
}